sycl/doc/GetStartedGuide.md

```diff
@@ -278,6 +276,50 @@ command:
     c:\oclcpu_rt_<cpu_version>\install.bat c:\tbb_<tbb_version>\tbb\bin\intel64\vc14
     ```
 
+### Obtain prerequisites for ahead of time (AOT) compilation
+
+[Ahead of time compilation](CompilerAndRuntimeDesign.md#ahead-of-time-aot-compilation)
+requires OpenCL ahead of time compiler available in `PATH`. There is
```

Comment:
```suggestion
requires ahead of time compiler available in `PATH`. There is
```
Does it work for Level Zero?

Reply:
No, ahead of time compilation is available for OpenCL only for now. I would prefer to keep "OpenCL" here to emphasize that. After adding support of other backends, this part can and should be changed.

Reply:
It would be good to have more explicit note somewhere on what backends support AOT.

Reply:
Sorry for confusion, Level Zero is supported in AOT mode, added which backends currently support AOT.